content/en/blog/2019/incorrect-sidecar-image-1.2.4/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Istio 1.2.4 sidecar image vulnerability
+description: An erroneus 1.2.4 sidecar image was available due to a faulty release operation.
+publishdate: 2019-09-10
+attribution: The Istio Team
+keywords: [community,blog,security]
+---
+To the Istio’s user community,
+
+**TL;DR**: For the period between Aug 23rd 2019 09:16PM PST and Sep 6th 2019 09:26AM PST a Docker image shipped as Istio `proxyv2` 1.2.4 (c.f. [https://hub.docker.com/r/istio/proxyv2](https://hub.docker.com/r/istio/proxyv2) ) contained a faulty version of the proxy against the security bugs [ISTIO-SECURITY-2019-003 and ISTIO-SECURITY-2019-004](/blog/2019/istio-security-003-004/).
```

Comment:
Remove the `**&TL;DR**: `, it's not necessary.

Reply:
```suggestion
For the period between Aug 23rd 2019 09:16PM PST and Sep 6th 2019 09:26AM PST a Docker image shipped as Istio `proxyv2` 1.2.4 (c.f. [https://hub.docker.com/r/istio/proxyv2](https://hub.docker.com/r/istio/proxyv2) ) contained a faulty version of the proxy against the security bugs [ISTIO-SECURITY-2019-003 and ISTIO-SECURITY-2019-004](/blog/2019/istio-security-003-004/).
```